src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -66,11 +69,21 @@ const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
     reset,
   } = useFormContext();
 
+  const formValues = getValues();
+
   const hasPermissions = useHasActionPermissions();
   const hasNoDecisionMethods = useHasNoDecisionMethods();
+  const hasEnoughMembersWithPermissions = useHasEnoughMembersWithPermissions({
+    decisionMethod: formValues[DECISION_METHOD_FIELD_NAME],
+    selectedAction: formValues[ACTION_TYPE_FIELD_NAME],
+    createdIn: formValues[CREATED_IN_FIELD_NAME],
+  });
 
   const isSubmitDisabled =
-    !selectedAction || hasPermissions === false || hasNoDecisionMethods;
+    !selectedAction ||
+    hasPermissions === false ||
+    !hasEnoughMembersWithPermissions ||
+    hasNoDecisionMethods;
```

Comment:
Checked this, but we are explicitly checking for is hasPermissions is false (it can be either true, false or undefined, and true and undefined are both fine, but false is not)